sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfig.java

```diff
@@ -82,4 +99,26 @@ public boolean isDefault() {
     public boolean continueOnInitError() {
         return continueOnInitError;
     }
+
+    /**
```

Comment:
Why do we need the static factory method when we have a builder? This seems inconsistent - and will be turned down during public API review process as well if I am not missing something and there is a good reason for this?

Reply:
As discussed in the earlier meeting, this was added to abstract the throughput and threshold configuration for customers who only want to use the priority based throttling feature.

Reply:
This logic is moved to builder